sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetFileFormat.scala

```diff
@@ -273,6 +293,31 @@ class GpuParquetWriter(
               new GpuColumnVector(DataTypes.TimestampType, withResource(cv.getBase()) { v =>
                 v.castTo(DType.TIMESTAMP_MILLISECONDS)
               })
+            case DataTypes.TimestampType
+              if outputTimestampType == ParquetOutputTimestampType.INT96.toString =>
+                withResource(Scalar.fromLong(Long.MaxValue / 1000)) { upper =>
+                  withResource(Scalar.fromLong(Long.MinValue / 1000)) { lower =>
+                    withResource(cv.getBase().bitCastTo(DType.INT64).greaterOrEqualTo(upper)) {
```

Comment:
bitCastTo returns a ColumnView that still has to be closed. You cannot chain them like this.

Reply:
Sorry, I keep forgetting the ColumnView is also a CPP side object and important to be closed